### PR TITLE
Adding Multi Cloud Testing Docs

### PR DESCRIPTION
**Pull request by deepakraj1997**

### CODE REVIEW

**Review by kaovilai — APPROVED** [review]

File: docs/developer/testing/ MULTI_CLOUD_TESTING_UPDATES.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# Multi Cloud Test Suite Updates for Backup / Restore cases 
+
+### Overview
+Running backup/restore test cases with cloud-provider specific snapshots enabled.
+
+The Test Suite now takes different env variables related to the cloud as flags and create the VSLs based on that cloud provider. Validating the backup / restore test cases on the same.
+
+### How multiple profiles are enabled to support Backup / Restore.
+In ideal cases, both the credentials / profile for BSL and VSL would be the same and usually we dont mention the separate creds for them, but this is different in CI environments. In CI environment, the cluster is provisioned in CI Cloud / Cluster which we dont have access to, hence supporting volume backup using our credentials which is mounted in CI environment is not a valid option. Hence we are using different methods to support these.
```
Comment (weshayutin) [inline docs/developer/testing/ MULTI_CLOUD_TESTING_UPDATES.md:9]: s/In CI environment/In a CI environment/
s//In CI environment/In the prow CI environment/

Comment (weshayutin) [reply docs/developer/testing/ MULTI_CLOUD_TESTING_UPDATES.md:9]: nit.. fix.." the cluster is provisioned in CI Cloud / Cluster"

Comment (weshayutin) [reply docs/developer/testing/ MULTI_CLOUD_TESTING_UPDATES.md:9]: nit.. mounted in CI environment

File: docs/developer/testing/ MULTI_CLOUD_TESTING_UPDATES.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+#### AWS Multi Profile Support
+
+The CI Cloud cred is present at this location in CI Environment,
```
Comment (weshayutin) [inline docs/developer/testing/ MULTI_CLOUD_TESTING_UPDATES.md:13]: nit.. Cred or cred ?  How about credentials?

File: docs/developer/testing/ MULTI_CLOUD_TESTING_UPDATES.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+`/var/run/oadp-credentials/new-aws-credentials`
+
+Here since they are two profiles, we are using the concept of credentialsFile in BSL config ref: https://github.com/vmware-tanzu/velero/issues/3428.
+
+We are also mounting credentials here: 
+https://github.com/deepakraj1997/oadp-operator/blob/multi-cloud-e2e-master-rebase/pkg/credentials/credentials.go#L38
+
+#### GCP
+
+The CI Cloud cred is present at this location in CI Environment,
```
Comment (weshayutin) [inline docs/developer/testing/ MULTI_CLOUD_TESTING_UPDATES.md:26]: nit.. cred / Cred

File: docs/developer/testing/ MULTI_CLOUD_TESTING_UPDATES.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+`/var/run/secrets/ci.openshift.io/cluster-profile/gce.json` 
+
+Our Cloud Cred used for BSL is present at this location.
+`/var/run/oadp-credentials/gcp-credentials`
+
+Here since they are two different credentials and not profiles, we are using the concept of credentialsFile in BSL config ref: https://github.com/vmware-tanzu/velero/issues/3430.
+
+We are also mounting credentials here: 
+https://github.com/deepakraj1997/oadp-operator/blob/multi-cloud-e2e-master-rebase/pkg/credentials/credentials.go#L48
```
Comment (weshayutin) [inline docs/developer/testing/ MULTI_CLOUD_TESTING_UPDATES.md:35]: create a MD link?

**Review by weshayutin — COMMENTED** [review]
THANK YOU!! for writing this up.
One more pass for grammer and readability would be a ++

nits.. mentioned in other comments

**Comment by weshayutin** [comment]
gnome-shell-screenshot-24av2y

seems like "VSL_REGION - the region" etc.. misrendered

**Comment by deepakraj1997** [comment]
Just copied from hackmd but its rendering a bit different in GitHub. Fixed review comments.

**Review by dymurray — APPROVED** [review]

**Pull request merged**